src/rdkafka_request.c

```diff
@@ -2283,9 +2284,11 @@ void rd_kafka_ConsumerGroupHeartbeatRequest(
         int16_t ApiVersion = 0;
         int features;
         size_t rkbuf_size = 0;
+        rd_kafkap_str_t *subscribed_topic_regex_to_send =
```

Comment:
Here it's possible it sends a NULL regex. We have to make sure that when we have a subscription to send we send the topic list even if empty and an empty string as the subscription regex. We have to do this otherwise with NULL it keeps the previous topic list or the previous subscription.